cluster/examples/kubernetes/ceph/csi/rbac/rbd/csi-provisioner-rbac.yaml

```diff
@@ -57,3 +54,32 @@ roleRef:
   kind: ClusterRole
   name: rbd-external-provisioner-runner
   apiGroup: rbac.authorization.k8s.io
+
+---
+kind: Role
```

Comment:
Are the Role definitions for `rbd-external-provisioner-cfg` and `cephfs-external-provisioner-cfg` the same? If so, could we define a common role that both provisioners would bind to their service account?

Reply:
But it's fine if we delay this refactoring until the csi driver is fully integrated with operator.yaml

Reply:
@travisnc, both are similar the reason I separated it because one is for cephfs and one is for RBD, sure I will do it later once CSI becomes stable

Reply:
@travisn I looked into having a single Role for both cephfs and rbd. currently, rbd-external-provisioner-cfg and cephfs-external-provisioner-cfg are binds to a different service account,
having a single service account is not possible as of now.